src/test/java/teammates/test/cases/ui/InstructorEditInstructorFeedbackSaveActionTest.java

```diff
@@ -313,7 +313,8 @@ private void testSubmitResponseForInvalidQuestion() throws Exception {
 
         gaeSimulation.loginAsInstructor(instructor.googleId);
 
-        ______TS("Failure case: submit response for question in session, but should not be editable by instructor (unable to see recipient)");
+        ______TS("Failure case: submit response for question in session, but should not be editable by instructor"
+                 + " (unable to see recipient)");
```

Comment:
Maybe move the whitespace in the string to the end of the string in the previous line ie. `...editable by instructor "` and `"(unable to see...` to be consistent with previous occurrences.